pyfolio/tests/test_perf_attrib.py

```diff
@@ -75,124 +75,6 @@ def mock_transactions_from_positions(positions):
-        )
-
-        expected_perf_attrib_output = pd.DataFrame(
-            index=dts,
-            columns=['risk_factor1', 'risk_factor2', 'common_returns',
-                     'specific_returns', 'total_returns'],
-            data={'risk_factor1': [0.025, 0.025],
-                  'risk_factor2': [0.025, 0.025],
-                  'common_returns': [0.05, 0.05],
-                  'specific_returns': [0.05, 0.05],
-                  'total_returns': returns}
-        )
-
-        expected_exposures_portfolio = pd.DataFrame(
-            index=dts,
-            columns=['risk_factor1', 'risk_factor2'],
-            data={'risk_factor1': [0.25, 0.25],
-                  'risk_factor2': [0.25, 0.25]}
-        )
-
-        exposures_portfolio, perf_attrib_output = perf_attrib(returns,
-                                                              positions,
-                                                              factor_returns,
-                                                              factor_loadings)
-
-        pd.util.testing.assert_frame_equal(expected_perf_attrib_output,
-                                           perf_attrib_output)
-
-        pd.util.testing.assert_frame_equal(expected_exposures_portfolio,
-                                           exposures_portfolio)
-
-        # test long and short positions
-        positions = pd.DataFrame(index=dts,
-                                 data={'stock1': [20, 20],
-                                       'stock2': [-20, -20],
-                                       'cash': [20, 20]})
-
-        exposures_portfolio, perf_attrib_output = perf_attrib(returns,
-                                                              positions,
-                                                              factor_returns,
-                                                              factor_loadings)
-
-        expected_perf_attrib_output = pd.DataFrame(
-            index=dts,
-            columns=['risk_factor1', 'risk_factor2', 'common_returns',
-                     'specific_returns', 'total_returns'],
-            data={'risk_factor1': [0.0, 0.0],
-                  'risk_factor2': [0.0, 0.0],
-                  'common_returns': [0.0, 0.0],
-                  'specific_returns': [0.1, 0.1],
-                  'total_returns': returns}
-        )
-
-        expected_exposures_portfolio = pd.DataFrame(
-            index=dts,
-            columns=['risk_factor1', 'risk_factor2'],
-            data={'risk_factor1': [0.0, 0.0],
-                  'risk_factor2': [0.0, 0.0]}
-        )
-
-        pd.util.testing.assert_frame_equal(expected_perf_attrib_output,
-                                           perf_attrib_output)
-
-        pd.util.testing.assert_frame_equal(expected_exposures_portfolio,
-                                           exposures_portfolio)
-
-        perf_attrib_summary, exposures_summary = create_perf_attrib_stats(
```

Comment:
Is `create_perf_attrib_stats` tested somewhere else?  This seems important to still have coverage for.

Reply:
the [test below this tests `create_perf_attrib_stats`](https://github.com/quantopian/pyfolio/pull/514/files/9f1a616a56ba433978a4281dd51e22ea6cc5af40#diff-e1d9a0a8c573c0639e3b1ffcf47fedb0R116). Should I keep the coverage here?

Reply:
Do you think the test below covers all the functionality we care to test for that function?